libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostStorage.m

```diff
@@ -74,6 +74,13 @@ - (id)init {
             }];
         }
         
+        //Load from info.plist
+        if([[NSBundle mainBundle] objectForInfoDictionaryKey:@"SFDCOAuthLoginHost"]) {
```

Comment:
Currently, we put all the login hosts in a NSMutableArray. Once they are added in there we pick the login host at the zeroth index to be the default host(Technically we can choose and pick any host at any index, but the default is zeroth index). Thats the reason why the MDM logic is first, then the info.plist. I do agree that once MDM is enforced we no longer need the other hosts in the NSMutable Array.